components/HTTP/src/Websocket.cpp

```diff
@@ -22,6 +22,37 @@ Websocket::Websocket()
     g_check_connections.start(NULL);
 }
 
+ /**
+     * @brief received encrypted message by radio
+     * @param data_received encrypted message
+*/
+static void receiveFromUart(void* data_received, void* length)
+{
+        char payload[500];
+        esp_err_t err;
+        fwrite(data_received, (size_t)length, 1, stdout);
```

Comment:
Please use ESP-IDF log functions, or hexdump log functions for this purpose so this can be deactivated when unused.